<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>The Computer Scientist DB</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>People</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="labelPeopleFilter">
            <property name="text">
             <string>Filter by:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="dropDownPeopleFilter">
            <item>
             <property name="text">
              <string>name</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>gender</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>birth</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>death</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditPeopleFilter"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tablePeople"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="lineEditPeopleName">
            <property name="minimumSize">
             <size>
              <width>280</width>
              <height>0</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="dropDownPeopleGender">
            <property name="editable">
             <bool>false</bool>
            </property>
            <property name="currentText">
             <string>select</string>
            </property>
            <item>
             <property name="text">
              <string>select</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>female</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>male</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditPeopleBirth">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>Year of birth</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditPeopleDeath">
            <property name="placeholderText">
             <string>Year of death</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="buttonPeopleAdd">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonPeopleDelete">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Delete selected</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>//label for error messages later</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Computers</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="labelComputersFilter">
            <property name="text">
             <string>Filter by:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="dropDownComputersFilter">
            <item>
             <property name="text">
              <string>name</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>type</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>created</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>built</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditComputersFilter"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableComputer"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="lineEditComputerName">
            <property name="minimumSize">
             <size>
              <width>280</width>
              <height>0</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditComputerType">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>Type</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditComputerCreated">
            <property name="placeholderText">
             <string>Year created</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="dropDownComputerBuilt">
            <property name="editable">
             <bool>false</bool>
            </property>
            <property name="currentText">
             <string>Built?</string>
            </property>
            <item>
             <property name="text">
              <string>Built?</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Yes</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>No</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="buttonComputerAdd">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonComputerDelete">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Delete selected</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="labelComputerError">
          <property name="text">
           <string>//label for error messages later</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>637</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
